dds/DCPS/ReactorTask.h

```diff
@@ -30,14 +30,22 @@ OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
 namespace OpenDDS {
 namespace DCPS {
 
+// thread status reporting forward
+struct ThreadStatus {
+  ACE_Recursive_Thread_Mutex lock;
+  OPENDDS_MAP(unsigned long, ACE_Time_Value) map;
```

Comment:
Please avoid dealing with `ACE_Time_Value`s unless you're dealing directly with ACE.